amazon-kinesis-client/src/test/java/software/amazon/kinesis/lifecycle/ShardConsumerTest.java

```diff
@@ -323,7 +322,7 @@ public void testLeaseLossIsNonBlocking() throws Exception {
         log.debug("Release processing task interlock");
         awaitAndResetBarrier(processingTaskInterlock);
 
-        while(!consumer.isShutdown()) {
+        while (!consumer.isShutdown()) {
```

Comment:
This is from the formatter. 

(2 cents) We really should either fully trust the formatter, fully automate it (format on save going forward, and reformat the codebase to ensure its consistant at a baseline), or be less concerned about whitespaces.   

Manually picking and choosing the what gets formatted is a waste of time.

Reply:
You should try to see why your formatter is running on the entire file all the time. We want to avoid white space changes. It is better to run the formatter only on vcs changes